bootstrapper-maven-plugin/src/main/resources/templates/CustomResource.java

```diff
@@ -0,0 +1,11 @@
+package {{groupId}};
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.Version;
+
+@Group("{{groupId}}")
+@Version("v1")
+public class {{artifactClassId}}CustomResource extends CustomResource<{{artifactClassId}}Spec,{{artifactClassId}}Status> implements Namespaced {
```

Comment:
This is going to use the `crd-generator` by default, should we have a separate template for getting started with the `java-gen`? Or are we good only with this approach?

Reply:
Good question, atm I'm aiming for an MVP version, we can extend it later.